includes/services.inc.php

```diff
@@ -108,13 +109,66 @@ function delete_service($service = null)
 function discover_service($device, $service)
 {
     if (! dbFetchCell('SELECT COUNT(service_id) FROM `services` WHERE `service_type`= ? AND `device_id` = ?', [$service, $device['device_id']])) {
-        add_service($device, $service, "(Auto discovered) $service");
+        add_service($device, $service, "$service Monitoring (Auto Discovered)", null, null, 0, 0, 0, "AUTO: $service", 0);
         log_event('Autodiscovered service: type ' . mres($service), $device, 'service', 2);
         echo '+';
     }
     echo "$service ";
 }
 
+function delete_service_template($service_template = null)
+{
+    if (! is_numeric($service_template)) {
+        return false;
+    }
+    remove_service_template($service_template);
+
+    return dbDelete('service_templates', '`id` =  ?', [$service_template]);
+}
+
+function discover_service_template($service_template = null)
+{
+    $services_template = ServiceTemplate::getServiceTemplate($service_template);
+    $status = null;
+    foreach (DB::table('device_group_device')->where('device_group_id', $services_template[0]['device_group_id'])->pluck('device_id') as $device) {
```

Comment:
Please use eloquent queries

Reply:
I've updated this to what i think is correct. :)